lib/google_drive/worksheet.rb

```diff
@@ -10,6 +10,21 @@
 require 'google_drive/list'
 
 module GoogleDrive
+  # A few default color instances. Google API reference:
+  # https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets#color
+  RED = Google::Apis::SheetsV4::Color.new(red: 1.0)
+  DARK_RED_1 = Google::Apis::SheetsV4::Color.new(red: 0.8)
```

Comment:
These are some of the standard colors in the Google palette.  It's convenient to have them when you're trying to match colors that users are already accustomed to seeing, but if you'd rather not have them in this API I'm good with that.  I can always just keep them in the project I'm working on that uses this.

FYI, here's a list of all the colors in the google palette: https://github.com/google/google-api-ruby-client/blob/master/samples/cli/lib/samples/sheets.rb

Reply:
Oh I see, it would be OK then. Where is the list of all colors? I couldn't find it in your link. It would be nicer to include all colors, but maybe we can add them later if needed.

Also can you put them in its own module, like GoogleDrive::Worksheet::Colors::RED?

Reply:
Also add a comment to mention that the colors match those available in the web UI.

Reply:
Whoops -- copy'n'pasted the wrong link.  Updated it now.

Reply:
Thanks!